data/json/mutations.json

```diff
@@ -413,10 +422,14 @@
         "id" : "INCONSPICUOUS",
         "name" : "Inconspicuous",
         "points" : 1,
-        "description" : "While sleeping or staying still, it is less likely that monsters will wander close to you.",
+        "description" : "You blend into the surroundings better than others. Monsters will notice you from shorter distances.",
```

Comment:
Between sentences, there should be two spaces, instead of one; in this case, `...others.  Monsters...`

Reply:
thanks, didn't realize that